.ado/templates/discover-google-test-adapter.yml

```diff
@@ -0,0 +1,12 @@
+steps:
```

Comment:
It seems like overkill to create a full YAML template only to run a few-lines PowerShell script.

Wouldn't it be better to use an actual PowerShell script and invoke it in a task/step?

That would also make it parameterizable, so it works for locating other support DLLs in the VS installation and not only the GTest adapter.

Reply:
I'm just backporting what we're using in 0.66, 0.67, and main, not creating a whole new solution.